backend/inspirehep/records/api/mixins.py

```diff
@@ -233,3 +234,73 @@ def get_newest_linked_conferences_uuid(self):
             pids_changed = set.symmetric_difference(set(pids_latest), pids_previous)
 
         return list(self.get_records_ids_by_pids(list(pids_changed)))
+
+
+class InstitutionPapersMixin:
+    def clean_institution_literature_relations(self):
+        InstitutionLiterature.query.filter_by(literature_uuid=self.id).delete()
+
+    def create_institution_relations(self):
+        institutions = self.get_linked_records_from_field("authors.affiliations.record")
+        institution_literature_relations_waiting_for_commit = []
+
+        for institution in institutions:
+            if institution.get("deleted") is not True:
```

Comment:
```suggestion
            if institution.get("deleted", False) is not True:
```

Reply:
We should have ``deleted`` property in base IMO, we're using it everywhere (not for this PR though)

Reply:
but in this case, I don't necessarily need the default False